src/Security/Model/Organization.php

```diff
@@ -53,7 +53,7 @@ public function getUsername()
 
     public function getUserIdentifier(): string
     {
-        return $this->getUsername();
```

Comment:
Why did you make this change? I suppose we return the `Organization` alias here now, instead of the userIdentifier?

Reply:
`getUsername` is deprecated in symfony, phpstan extension for symfony now report it. In fact, `getUsername` return also alias so it's same